engine/api/services.go

```diff
@@ -36,7 +36,10 @@ func (api *API) getServiceHandler() service.Handler {
 		// Try to load from DB
 		var srvs []sdk.Service
 		var err error
-		if isAdmin(ctx) || isMaintainer(ctx) {
+		if isAdmin(ctx) {
```

Comment:
Maybe we can only check isMaintainer and remove tracking ?

Reply:
you are right, PU updated